src/Tests/Project/ToolRunner.test.ts

```diff
@@ -104,12 +104,10 @@ suite('Project', function() {
         for (let i = 0; i < simultaneousTrials; i++) {
           let args = new ToolArgs();
           args.push('1');  // 1 sec
-          try {
-            toolRunner.getRunner('sleep', 'sleep', args, '.');
```

Comment:
:bell: **Details** :bell: 

`await` or `Promise.prototype.catch` should be used

Reply:
This err is raised before returning promise.

https://github.com/Samsung/ONE-vscode/blob/c861c744a178975d07742a67c76624bb0e781335/src/Project/ToolRunner.ts#L160-L170

Reply:
Error raising is expected and that's what this test want to check.
My intention was catch that error in `catch()` but not caught...